oresat_star_tracker/camera.py

```diff
@@ -34,14 +44,10 @@ def read_image_size(self):
 
     def read_context_setting(self, name: str) -> int:
         """'Read a context setting."""
-
         context_path = "/sys/devices/platform/prucam/context_settings"
-        try:
-            with open(f"{context_path}/{name}", "r") as f:
-                value = int(f.read())
-        except FileNotFoundError:
-            raise CameraError(f"no sysfs attribute {name} for camera")
-        return value
+        with open(f"{context_path}/{name}", "r") as f:
+            value = int(f.read())
+            return value
```

Comment:
don't return inside of a with block

Reply:
Why not? It'll do the right thing, the resource will be cleaned up (the file closed) no matter how you leave the block, `return` included. Returning inside a with block is even called out in the relevant PEP as a reason why you'd want to use it.